src/components/Calculator.js

```diff
@@ -1,35 +1,47 @@
 import React from 'react';
+import calculate from '../logic/calculate';
 
 class Calculator extends React.Component {
```

Comment:
Division functionality is not present in app. Kindly make sure that user is able to divide two numbers.
![devision](https://user-images.githubusercontent.com/35504149/148590645-fd5959a4-e092-47e1-8a9a-4041e61f61a3.png)